pkg/server/routes.go

```diff
@@ -90,4 +90,8 @@ func (s *Server) routes(
 		}
 		api.Handle("/system_intakes", systemIntakesHandler.Handle())
 	}
+
+	s.router.PathPrefix("/").Handler(handlers.CatchAllHandler{
```

Comment:
Mux also has `api.NotFoundHandler`. Not saying we should use it, as using the more explicit path setting does the trick and is more standard, but it's an option if for whatever reason there's an issue.

Reply:
we considered that, but wanted the extra logging of what url was attempted.